app/assets/javascripts/discourse/app/lib/public-js-versions.js

```diff
@@ -2,11 +2,13 @@
 // Update it by running `rake javascript:update`
 
 export const PUBLIC_JS_VERSIONS = {
-  ace: "1.4.12",
-  "Chart.min.js": "2.9.3",
-  "chartjs-plugin-datalabels.min.js": "0.7.0",
-  "jquery.magnific-popup.min.js": "1.1.0",
-  "pikaday.js": "1.8.0",
-  "spectrum.js": "1.8.0",
-  workbox: "4.3.1",
+  "ace/ace.js": "ace.js/1.4.12/ace.js",
+  "Chart.min.js": "chart.js/2.9.3/Chart.min.js",
+  "chartjs-plugin-datalabels.min.js":
+    "chartjs-plugin-datalabels/0.7.0/chartjs-plugin-datalabels.min.js",
+  "jquery.magnific-popup.min.js":
+    "magnific-popup/1.1.0/jquery.magnific-popup.min.js",
+  "pikaday.js": "pikaday/1.8.0/pikaday.js",
+  "spectrum.js": "spectrum-colorpicker/1.8.0/spectrum.js",
+  "spectrum.css": "spectrum-colorpicker/1.8.0/spectrum.css",
```

Comment:
I'm noticing that workbox is missing, both here, and in the number-versioned folder. is that by design?

Reply:
workbox doesn't appear to be loaded via the loadScript function, and so wouldn't be picked up by these changes.

We absolutely could have a versioned directory for workbox written out by the `javascript:update` rake task, but it would need additional work to ensure it's correctly referenced. I'd recommend doing this as a separate commit.